### PR TITLE
fix(core): TLS handshake might occasionally get stuck on MacOS

### PR DESCRIPTION
**Pull request by jerrinot**

### CODE REVIEW

File: core/src/main/java/io/questdb/network/IODispatcherOsx.java [hunk 1]
```diff
@@ -322,12 +325,20 @@ private boolean processRegistrations(long timestamp) {
                 keventWriter.readFD(suspendEvent.getFd(), eventId);
             }
 
-            if (operation == IOOperation.READ || context.getSocket().wantsTlsRead()) {
```
Comment (puzpuzpuz) [inline core/src/main/java/io/questdb/network/IODispatcherOsx.java:325]: nit: the `if` blocks order change here is irrelevant and not required for the fix.

**Review by puzpuzpuz — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:disappointed: **fail** : 0 / 6 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/network/IODispatcherOsx.java|0|6|00.00%|

File: core/src/main/java/io/questdb/network/IODispatcherOsx.java (hunk 1 above)
Comment (jerrinot) [reply core/src/main/java/io/questdb/network/IODispatcherOsx.java:325]: I am not sure I am following. can you elaborate please?

Comment (puzpuzpuz) [reply core/src/main/java/io/questdb/network/IODispatcherOsx.java:325]: Sorry for the confusion, I misread the changes in `processRegistrations`. Both changes are relevant.

**Pull request merged**